tools/bumbershoot/idpassemble/idpassemble.xml

```diff
@@ -0,0 +1,122 @@
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-mm.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-msgf.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-msgf.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-cm.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803-cm.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+        <test>
+            <param name="input" value="../../idpqonvert/test-data/201208-378803-mm.idpDB,../../idpqonvert/test-data/201208-378803-msgf.idpDB,../../idpqonvert/test-data/201208-378803-cm.idpDB" />
+            <param name="max_fdr_score" value="0.05" />
+            <param name="filter_at_gene_level_condition.min_distinct_peptides" value="2" />>
+            <param name="filter_at_gene_level_condition.min_spectra" value="2" />
+            <param name="filter_at_gene_level_condition.min_additional_peptides" value="1" />
+            <param name="min_spectra_per_distinct_match" value="1" />
+            <param name="min_spectra_per_distinct_peptide" value="1" />
+            <param name="max_protein_groups_per_peptide" value="10" />
+            <output name="output" file="../../idpassemble/test-data/201208-378803.idpDB" compare="sim_size" delta_size="500000" />
+        </test>
+    </tests>
+    <help>
+<![CDATA[
+**What it does**
+
+Merges and filters one or more IDPicker 3 idpDB files into a combined idpDB file. Protein assembly (e.g. parsimony) is conducted on the combined set of proteins.
```

Comment:
If you have a link to more documentation or some images, detailed explanations ....